### PR TITLE
📖  Add v1alpha4 note to readme

### PR DESCRIPTION
**Pull request by furkatgofurov7**
**What this PR does / why we need it**:
Adds a note regarding the v1alpha4 version of CAPM3 to readme

### CODE REVIEW

**Review by maelk — APPROVED** [review]

**Comment by furkatgofurov7** [comment]
Should we wait for CI to be back to run the test again, @maelk ?

**Comment by maelk** [comment]
yes, hopefully things will be fixed by EOD

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ For more information about this actuator and related repositories, see
 |---------------|---------------------|---------|
 | v1alpha2      | v1alpha2            | v0.2.X  |
 | v1alpha3      | v1alpha3            | v0.3.X  |
+| v1alpha4      | v1alpha3            | v0.4.X  |
```
Comment (Jaakko-Os) [inline README.md:18]: 'Release', is that for CAPM3 or CAPI?

**Review by Jaakko-Os — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (furkatgofurov7) [reply README.md:18]: @Jaakko-Os My understanding was CAPM3

Comment (Jaakko-Os) [reply README.md:18]: It would be nice to have unambiguous documentation. My proposal is to add CAPM3 to version field.

Comment (furkatgofurov7) [reply README.md:18]: @Jaakko-Os do you mean `Release field`? (i.e CAPM3 Release)

Comment (Jaakko-Os) [reply README.md:18]: yes, sorry for my typo

**Review by Jaakko-Os — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (furkatgofurov7) [reply README.md]: Done, thanks!

**Pull request merged**